bluemira/codes/process/api.py

```diff
@@ -80,9 +79,42 @@ def __init__(self, filename):
     PROCESS_DICT = get_dicts()
 
 
-DEFAULT_INDAT = os.path.join(
-    get_bluemira_path("codes/process"), "PROCESS_DEFAULT_IN.DAT"
-)
+@dataclass
+class INVariable:
```

Comment:
Could/should this be a private class?

```suggestion
class _INVariable:
```